<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDataset</class>
 <widget class="QDialog" name="CreateDataset">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mass: Load Data</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
    <item>
     <widget class="QLabel" name="label">
      <property name="toolTip">
       <string>Whether to load indicated pulse files</string>
      </property>
      <property name="text">
       <string>Select one example file for pulse and/or noise data:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="file_selection_layout" rowstretch="0,0,0,0,0" columnstretch="0,1,0" rowminimumheight="0,0,0,0,0" columnminimumwidth="65,400,75">
      <item row="0" column="1">
       <widget class="QLineEdit" name="pulse_file_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="use_pulses">
        <property name="toolTip">
         <string>Whether to load indicated pulse files.</string>
        </property>
        <property name="text">
         <string>Pulses</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="use_noise">
        <property name="toolTip">
         <string>Whether to load indicated noise files</string>
        </property>
        <property name="text">
         <string>Noise</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="noise_file_edit">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="choose_pulse_file">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Choose the template pulse file name.</string>
        </property>
        <property name="text">
         <string>choose</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="choose_noise_file">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Choose the template pulse file name.</string>
        </property>
        <property name="text">
         <string>choose</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="chan_selection_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="chan_selection_layout" rowstretch="0" columnstretch="0"/>
      </widget>
      <widget class="QLabel" name="chan_selection_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>0</y>
         <width>551</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose channels to load</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>16</y>
         <width>111</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="chan_selection_controls">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="check_all_chan">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Select all channels on disk</string>
          </property>
          <property name="text">
           <string>All chan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="check_default_chan">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Select all channels but the disabled ones.</string>
          </property>
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="check_no_chan">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Unselect all channels</string>
          </property>
          <property name="text">
           <string>No chan</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="take_action">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="summarize_on_load">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Whether to run method &lt;span style=&quot; font-style:italic;&quot;&gt;summarize_data&lt;/span&gt;() on dataset &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;before returning it.  (Can be convenient but slower.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Summarize on load</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="ok_cancel_buttons">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>use_pulses</tabstop>
  <tabstop>pulse_file_edit</tabstop>
  <tabstop>choose_pulse_file</tabstop>
  <tabstop>use_noise</tabstop>
  <tabstop>noise_file_edit</tabstop>
  <tabstop>choose_noise_file</tabstop>
  <tabstop>check_all_chan</tabstop>
  <tabstop>check_default_chan</tabstop>
  <tabstop>check_no_chan</tabstop>
  <tabstop>summarize_on_load</tabstop>
  <tabstop>ok_cancel_buttons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok_cancel_buttons</sender>
   <signal>accepted()</signal>
   <receiver>CreateDataset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_cancel_buttons</sender>
   <signal>rejected()</signal>
   <receiver>CreateDataset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pulse_file_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateDataset</receiver>
   <slot>file_template_textEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noise_file_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateDataset</receiver>
   <slot>file_template_textEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>choose_file()</slot>
  <slot>file_template_textEdited(QString)</slot>
 </slots>
</ui>
